feature-engineering/categorical_imputer.md

```diff
@@ -0,0 +1,153 @@
+Impute missing categorical values using reference features.
```

Comment:
I'm reviewing this in isolation (I don't think I reviewed that one, or if I did, I was less thorough). Please feel free to keep things consistent (consistency is probably better than having better grammar in one place).